src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/IExtendedTemplateParameter.cs

```diff
@@ -1,11 +1,11 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using Microsoft.TemplateEngine.Abstractions;
 
 namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
 {
     public interface IExtendedTemplateParameter : ITemplateParameter
     {
-        string FileRename { get; }
+        //string FileRename { get; }
```

Comment:
I suppose that changes of public interfaces could be problematic. What if somebody uses them outside of dotnet/templating?